src/renderer/views/Search/Search.js

```diff
@@ -108,6 +110,15 @@ export default defineComponent({
     this.checkSearchCache(payload)
   },
   methods: {
+    updateSearchHistoryEntry: function () {
+      const persistentSearchHistoryPayload = {
+        _id: this.query,
+        lastUpdatedAt: new Date()
```

Comment:
As this field is just used for sorting, we might as well save it as a number directly.
```suggestion
        lastUpdatedAt: Date.now()
```